apps/src/templates/YourSchool.jsx

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
+import YourSchoolResources from './YourSchoolResources';
+import Notification from './Notification';
+import i18n from "@cdo/locale";
+
+const styles = {
+  description: {
+    marginBottom: 20
+  },
+};
 
 export default class YourSchool extends React.Component {
   render() {
     return (
       <div>
         <h1>
-          Bring Computer Science to your school district
+          {i18n.yourSchoolHeading()}
         </h1>
-        <h3>
-          Our vision is that every student in every school should have the opportunity to learn computer science.
-          Help us by letting us know how computer science is taught at your school and by pledging to do more.
+        <Notification
+          type="bullhorn"
+          notice="Something exciting happened"
+          details="Here's some more information about the exciting thing"
+          dismissible={false}
+          buttonText="Learn more"
+          buttonLink="/blog"
+          newWindow={true}
+          isRtl={false}
+        />
```

Comment:
What is ultimately going to populate this Notification?

Reply:
You don't think that's a good announcement 😜? My understanding is that this announcement will have information about the upcoming secret funding that isn't disclosed yet and link to a blog post about it.

Reply:
Mostly just curious about what the i18n story will be for it. "Learn more" there's probably an existing string for (and might not change). For the others, we'll just presumably want to add i18n strings when we have the final copy.

Reply:
Worth noting that it's good to not add i18n until we have the final string, as you imply, otherwise some languages might end up with translations of a placeholder string.